docs/no_peeking/mappings/individual.py

```diff
@@ -0,0 +1,114 @@
+"""
+This module maps Kids First participant to Phenopackets Individual (derived from FHIR Patient).
+Please visit https://aehrc.github.io/fhir-phenopackets-ig/StructureDefinition-Individual.html
+    for the detailed structure definition.
+"""
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+
+
+def karyotypic_sex(x):
+    """
+    http://ga4gh.org/fhir/phenopackets/CodeSystem/KaryotypicSex
+    """
+    if x == constants.GENDER.FEMALE:
+        return {
+            'system': 'http://ga4gh.org/fhir/phenopackets/CodeSystem/KaryotypicSex',
+            'code': 'XX', 
+            'display': 'Female'
+        }
+    elif x == constants.GENDER.MALE:
+        return {
+            'system': 'http://ga4gh.org/fhir/phenopackets/CodeSystem/KaryotypicSex',
+            'code': 'XY',
+            'display': 'Male'
+        }
+    elif x == constants.COMMON.OTHER:
+        return {
+            'system': 'http://ga4gh.org/fhir/phenopackets/CodeSystem/KaryotypicSex',
+            'code': 'OTHER_KARYOTYPE',
+            'display': 'None of the above types'
+        }
+    elif x == constants.COMMON.UNKNOWN:
+        return {
+            'system': 'http://ga4gh.org/fhir/phenopackets/CodeSystem/KaryotypicSex',
+            'code': 'UNKNOWN_KARYOTYPE',
+            'display': 'Untyped or inconclusive karyotyping'
+        }
+    else:
+        raise Exception('Unknown Individual KaryotypicSex')
+
+def taxonomy(x):
+    """
+    http://purl.obolibrary.org/obo/ncbitaxon.owl
+    """
+    if x == constants.SPECIES.DOG:
+        return {
+            'system': 'http://purl.obolibrary.org/obo/ncbitaxon.owl',
+            'code': 'NCBITaxon:9615',
+            'display': 'Canis lupus familiaris'
+        }
+    elif x == constants.SPECIES.HUMAN:
+        return {
+            'system': 'http://purl.obolibrary.org/obo/ncbitaxon.owl',
+            'code': 'NCBITaxon:9606',
+            'display': 'Homo sapiens'
+        }
+    else: 
+        raise Exception('Unknown Individual Taxonomy')
+
+def gender(x):
+    """
+    http://hl7.org/fhir/R4/valueset-administrative-gender.html
+    """
+    if x == constants.GENDER.FEMALE:
+        return 'female'
+    elif x == constants.GENDER.MALE:
+        return 'male'
+    elif x == constants.COMMON.OTHER:
+        return 'other'
+    elif x == constants.COMMON.UNKNOWN:
+        return 'unknown'
+    else:
+        raise Exception('Unknown Individual gender')
+
+
+individual = {
+    'resourceType': 'Patient',
+    'id': CONCEPT.PARTICIPANT.TARGET_SERVICE_ID,
+    'meta': {
+        'profile': ['http://ga4gh.fhir.phenopackets/StructureDefinition/Individual']
+    },
+    'extension': [
+        {
+            'text': 'KaryotypicSex',
```

Comment:
What's the 'text' field for? I'm guessing for documentation. It doesn't have a place in Extension.

Reply:
Agreed, there is no `text` field in extension but there is one in the CodeableConcept. Move the `text` field in the CodeableConcept like this `valueCodeableConcept.text`

Reply:
@fiendish will create a PR to this branch which fixes this issue!